abb_irb2400_support/launch/test_irb2400_12_155.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
+        <include file="$(find abb_irb2400_support)/launch/load_irb2400_12_155.launch" />
+	<param name="use_gui" value="true" />
+	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find industrial_robot_client)/config/robot_state_visualize.rviz" required="true" />
+</launch>
```

Comment:
Indentation, and please make `use_gui` a private parameter of `joint_state_publisher`.